smaug/python/ops/nn_ops.py

```diff
@@ -74,6 +74,66 @@ def compute_output_dim(input_dim, weight_dim, stride, padding):
       output_tensors_dims=[output_tensor_dims],
       output_tensor_layout=output_layout, params=params)[0]
 
+def depthwise_convolution(
```

Comment:
I would prefer that this is only enabled for the Reference backend, not for SMV (unless we properly implement an accelerated kernel for it). Can you first do a check for the backend? You can get this with `get_graph().backend()` (don't forget to check for `graph == None` too).